kafka-ui-react-app/src/components/Topics/Copy/Copy.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
```

Comment:
a question if Copy and New Components are the same thing with or without the data we should create a component to use in both page and write test suites in that component. ?

Like in New -> pass empty props and test the filling it.
Like in Copy -> check the already existing data in it and test the submit.

Reply:
I think that is a good idea. So there are two ways of doing it: 
-	one shared component for Create and Copy topic, which checks if there are some query params (but this logic could become more complicated with components and project grow)
-	another way to split the functionality of Create and Copy to appropriate pages or components, so they could expand without expanding the complexity

Which way we prefer?